src/Agent.Worker/Release/AgentUtilities.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker.Release
```

Comment:
This comes in the timeline record. Lot of times when we are debugging the issue on VSO we just lookup the timeline records. Also user would be aware what variables are available. It was easy for customer to lookup. In few occasions, customer directly reported that this variables is not set on this release. it was easy for us to get started debugging the issue. I would keep this code as is. If you plan to use this in build please move it to common place.